README.md

```diff
@@ -0,0 +1,6 @@
+# Repository forms
+
+[![Build Status](https://img.shields.io/travis/ezsystems/repository-forms.svg?style=flat-square)](https://travis-ci.org/ezsystems/repository-forms)
+[![Downloads](https://img.shields.io/packagist/dt/ezsystems/repository-forms.svg?style=flat-square)](https://packagist.org/packages/ezsystems/repository-forms)
+[![Latest stable](https://img.shields.io/packagist/v/ezsystems/repository-forms.svg?style=flat-square)](https://packagist.org/packages/ezsystems/repository-forms)
```

Comment:
I noticed that also in case of similar Solr PR. It seems to me that this is rather a https://shields.io bug, links with `/v` should point to Release, not Pre-Release (according to doc on their webpage). Not sure if we can do something about it, but @MarioBlazek - some research welcomed ;)